serverlessworkflow/modules/ROOT/pages/cloud/operator/developing-workflows.adoc

```diff
@@ -201,6 +208,23 @@ In order to remove the {product_name} Greeting application, you can execute the
 kubectl delete -f <your_file> -n <your_namespace>
 ----
 
+== Referencing resources in the workflow
```

Comment:
@ricardozanini Nit: I think we want to refer to **W**orkflow and not **w**orkflow throughout the docs?